ui/src/router/components/form/components/outcome_config/kafka/KafkaConfigPanel.js

```diff
@@ -1,15 +1,39 @@
 import React from "react";
 import { Panel } from "../../Panel";
-import { EuiFieldText, EuiForm, EuiFormRow, EuiSpacer } from "@elastic/eui";
+import {
+  EuiFieldText,
+  EuiForm,
+  EuiFormRow,
+  EuiLink,
+  EuiSpacer,
+  EuiSuperSelect
+} from "@elastic/eui";
 import { FormLabelWithToolTip } from "../../../../../../components/form/label_with_tooltip/FormLabelWithToolTip";
 import { useOnChangeHandler } from "../../../../../../components/form/hooks/useOnChangeHandler";
+import { resultLoggingConfig } from "../../../../../../config.js";
+
+const logSerializationFormatOptions = [
+  {
+    value: "json",
+    inputDisplay: "JSON"
+  },
+  {
+    value: "protobuf",
+    inputDisplay: "Protobuf",
+    helpText: "TuringResultLog.proto",
+    helpUrl: resultLoggingConfig.protoUrl
```

Comment:
Would it be a better idea to make `helpText` one of `string | ReactNode`? So you can specify the entire logic here as:
 ```js
helpText: resultLoggingConfig.protoUrl 
  ? (
    <EuiLink href={resultLoggingConfig.protoUrl} target="_blank" external>
            TuringResultLog.proto
    </EuiLink>
  ) : ""
```

and then use it at line 91 as: 
```js
   helpText={ (selectedFormat || {}).helpText }
```